ui/src/app/cohort-review/participant.model.ts

```diff
@@ -24,13 +39,15 @@ export class Participant implements ParticipantCohortStatus {
   /* Constructor & static methods */
   constructor(obj?: ParticipantCohortStatus) {
     if (obj) {
-      const {participantId, status, gender, race, ethnicity, birthDate} = obj;
+      const {participantId, status, gender, race, ethnicity, birthDate, deceased} = obj;
       this.id = participantId;
       this.status = status;
-      this.gender = gender;
+      this.gender = !!gender ? gender.charAt(0).toUpperCase() + gender.slice(1).toLowerCase()
+        : gender;
```

Comment:
This logic seems to be duplicated from above. Maybe call -> formattedGenderText above?